extensionsGallery-insider.json

```diff
@@ -2413,7 +2413,7 @@
 								},
 								{
 									"assetType": "Microsoft.VisualStudio.Services.Content.Details",
-									"source": "https://downloads.sentryone.com/downloads/ads/readme.0.9.7.md"
+									"source": "https://downloads.sentryone.com/downloads/ads/readme.0.9.8.md"
```

Comment:
I get an error when trying this link - has the new readme been uploaded yet?

Reply:
It has not yet. We typically publish the readme along with the extension. I can probably push up the readme if that would help.

Reply:
please publish it now and let me know, I will update the gallery files once I hear back from you.

Reply:
readme.0.9.8.md has been published to our CDN.

https://downloads.sentryone.com/downloads/ads/readme.0.9.8.md

Our release date for the extension is Monday 10/19.

Reply:
Ok, ping us when it's released and we can merge it in and publish the updated gallery at that time.

Reply:
Version 0.9.8 of the PE ADS Extension has been published to our portal.

https://extensions.sentryone.com/